controller-server/src/test/java/com/yahoo/vespa/hosted/controller/maintenance/MetricsReporterTest.java

```diff
@@ -300,15 +301,22 @@ public void test_nodes_failing_os_upgrade() {
             reporter.maintain();
             assertEquals(0, getNodesFailingOsUpgrade());
 
-            // 1/3 nodes upgrade within timeout
-            tester.configServer().setOsVersion(SystemApplication.tenantHost.id(), zone.getId(), version, 1);
-            tester.clock().advance(Duration.ofMinutes(30).plus(Duration.ofSeconds(1)));
+            // 2/6 nodes upgrade within timeout
+            tester.configServer().setOsVersion(SystemApplication.tenantHost.id(), zone.getId(), version, 2);
+            tester.clock().advance(Duration.ofMinutes(30 * 4 /* time allowance * node count */).plus(Duration.ofSeconds(1)));
```

Comment:
Ttwo have upgraded, so x3 + 1s should be enough to fail the rest, if I understand correctly?